inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -54,7 +55,8 @@ def simulate(
             resource_pool_id: Optional[UUID] = None,
             simulation_time: float = 10,  # ns
             integrator: Literal["md", "sd", "bd"] = "md",
-            nsteps_minim: int = 5000):
+            nsteps_minim: int = 5000,
+            visualized_section: str = "Protein-H"):
```

Comment:
It's weird to be setting visualization parameters in the scenario methods. How about calling this `output_trajectory_section`? This way you are defining the section of the trajectory to be computed, and not exactly what should/will be visualized...